bpf/lib/nodeport.h

```diff
@@ -674,7 +655,7 @@ int tail_nat_ipv64(struct __ctx_buff *ctx)
 declare_tailcall_if(__not(is_defined(IS_BPF_LXC)), CILIUM_CALL_IPV6_NODEPORT_NAT_INGRESS)
 int tail_nodeport_nat_ingress_ipv6(struct __ctx_buff *ctx)
 {
-	const bool nat_46x64 = ctx_load_meta(ctx, CB_NAT_46X64);
+	const bool nat_46x64 = nat46x64_cb_xlate(ctx);
```

Comment:
As discussed, pretty confident this can go away completely (just copy & pasted from the ingress/egress split).